gravitee-apim-console-webui/src/management/api/notifications/notifications-add-notification-dialog/notifications-add-notification-dialog.component.ts

```diff
@@ -13,11 +13,37 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component } from '@angular/core';
+import { Component, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+
+import { NotificationSettings } from '../../../../entities/notification/notificationSettings';
+
+export interface NotificationsAddNotificationDialogData {
+  notifier: NotificationSettings[];
+}
 
 @Component({
   selector: 'notifications-add-notification-dialog',
   template: require('./notifications-add-notification-dialog.component.html'),
   styles: [require('./notifications-add-notification-dialog.component.scss')],
 })
-export class NotificationsAddNotificationDialogComponent {}
+export class NotificationsAddNotificationDialogComponent {
+  notificationForm: FormGroup;
+  notifierList: NotificationSettings[];
+
+  constructor(
+    @Inject(MAT_DIALOG_DATA) dialogData: NotificationsAddNotificationDialogData,
+    public dialogRef: MatDialogRef<NotificationsAddNotificationDialogData>,
+  ) {
+    this.notifierList = dialogData.notifier;
+    this.notificationForm = new FormGroup({
+      name: new FormControl(null, [Validators.required]),
+      notifier: new FormControl(this.notifierList[0].id, [Validators.required]),
+    });
+  }
+
+  onSubmit() {
+    this.dialogRef.close(this.notificationForm.getRawValue());
```

Comment:
What do you think to add a Result type for this dialog ?
```
NotificationsAddNotificationDialogResult = NewNotification
```
so that we keep a naming convention 🤷‍♂️

Reply:
Done